frontend/tests/unit/components/layout/Tabs.spec.ts

```diff
@@ -32,23 +39,23 @@ describe('Tabs.vue', () => {
   });
 
   it('shows first tab content per default', () => {
-    const tabs = [{ template: 'foo' }, { template: 'bar' }, { template: 'baz' }];
+    const tabs = [{ template: '<foo />' }, { template: '<bar />' }, { template: '<baz />' }];
     const wrapper = createWrapper({ tabs });
     const tabContent = wrapper.get('[data-test="tab-content"]');
 
     expect(tabContent.html()).toContain('foo');
```

Comment:
Since it was my first time reading these tests it took me quite a while to understand why we are checking against a string `'foo'` here instead of trying to search for a component inside the html. 
To make these tests more understandable we could use `findComponent('foo').exists()` here for example.

The same goes for the other test cases where a similar pattern is used.

Reply:
sure can do so :+1:

Reply:
:heavy_check_mark: